src/Console/Command/SelfUpdateCommand.php

```diff
@@ -109,14 +110,31 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $tempFilename = basename($localFilename, '.phar').'-tmp.phar';
         $remoteFilename = sprintf('https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/%s/php-cs-fixer.phar', $remoteTag);
 
+        $details = null;
         try {
-            $copyResult = @copy($remoteFilename, $tempFilename);
-            if (false === $copyResult) {
-                $output->writeln(sprintf('<error>Unable to download new version %s from the server.</error>', $remoteTag));
+            $copyResult = copy($remoteFilename, $tempFilename);
```

Comment:
`@` was silenting the warnings you got when sth goes wrong that couldn't be catched